projects/Mallard/src/helpers/settings.ts

```diff
@@ -74,6 +74,7 @@ export interface DevSettings {
 
 interface UserSettings {
     hasOnboarded: boolean
+    weatherVisibility: 'shown' | 'hidden'
```

Comment:
I think we would only have a true/false situation with the weather?
```suggestion
    showWeather: boolean
```

Reply:
I'm fine to change for a boolean, but from experience, enum are more flexible in case any change needs to be done in the future, such as a three-way state

Reply:
I don't see how this can be a three way state?

Reply:
sure thing, I updated to a boolean! what do you think?